test/Microsoft.AspNet.Session.Tests/project.json

```diff
@@ -1,5 +1,6 @@
 {
     "dependencies": {
+        "Microsoft.AspNet.Http.Features": "1.0.0-*",
```

Comment:
You shouldn't need a direct Features dependency, Session should already depend on it.